renovate.json

```diff
@@ -27,5 +27,6 @@
       }
     }
   ],
-  "reviewers": ["ryanseddon", "jzempel", "austin94"]
+  "labels": ["PR: Internal :bouquet:"],
+  "reviewers": ["zendeskgarden/maintainers"]
```

Comment:
Does this work now? In the past, renovate was unable to notify groups (due to limited API permissions).

Reply:
> In the past, renovate was unable to notify groups

Didn't know this was an issue. I will revert.